pkg/plugin/manager.go

```diff
@@ -246,3 +255,62 @@ func (m *Manager) Install(url string) error {
 
 	return nil
 }
+
+func (m *Manager) InstallWallet() error {
+	var name = "Wallet"
+	var version = "v0.0.6"
+	var baseName = "thyra-plugin-wallet"
+	var baseUrl = "https://github.com/massalabs/"
+	var url = baseUrl + baseName + "/releases/download/" + version + "/" + baseName + "_"
```

Comment:
I think we should use `latest` to avoid having to update it all the time.
I don't think it's a too bad practice since, IMO, this code won't stay there. We will need something more robust to install plugins by default (we could imagine a config file listing the installed plugins, and if one plugin isn't installed, the plugin manager would install it automagically) 

```suggestion
func (m *Manager) InstallWallet() error {
	var name = "Wallet"
	var version = "latest"
	var baseName = "thyra-plugin-wallet"
	var baseUrl = "https://github.com/massalabs/"
	var url = baseUrl + baseName + "/releases/" + version + "/download/" + baseName + "_"
```

Which should give a link like:
https://github.com/massalabs/thyra-plugin-wallet/releases/latest/download/thyra-plugin-wallet_darwin-amd64.zip

Reply:
@SlnPons @gregLibert wdyt?

Reply:
Just to make it clear, I'm not saying we need to implement the "robust" version now. 

For now, I think the link we use to download the Wallet Plugin should point to `latest` to make sure we always have the latest version of the Wallet Plugin without having to do a release of Thyra each time a new release of the Wallet Plugin gets out

Reply:
the drawback of latest is that if latest is broken, then prod is broken and we don't know it (it happened with node manager)